test/blackbox-tests/test-cases/glob-deps-outside-directory.t

```diff
@@ -0,0 +1,20 @@
+Glob deps in a subdirectory that refer to a path outside that subdirectory
```

Comment:
Can this be moved to a separate PR?

Reply:
moved to https://github.com/ocaml/dune/pull/6332